code/modules/clothing/head/misc_vr.dm

```diff
@@ -172,3 +172,10 @@
 	icon = 'icons/inventory/head/item_vr.dmi'
 	default_worn_icon = 'icons/inventory/head/mob_vr.dmi'
 	icon_state = "tiny_tophat"
+
+/obj/item/clothing/head/halo
```

Comment:
// chompedits.

Reply:
Hopefully that was everything branded with the chomp logo.